### PR TITLE
Fix for HTTP v2

### LINKED ISSUE #1076: Undefined offset: 1 (enabled HTTP/2 support in the Curl library used by PHP)
https://github.com/facebook/php-graph-sdk/blob/3138249d85e8dc040f15bedd3581a3c616f49005/src/Facebook/Http/GraphRawResponse.php#L107

From today I am getting error like in title. I can see in logs this call:

`Facebook\Http\GraphRawResponse->setHttpResponseCodeFromHeader('HTTP/2 200 ')`

It doesn't match to regex in code: 
`|HTTP/\d\.\d\s+(\d+)\s+.*|`

The problem appeared after we enabled HTTP/2 support in the Curl library used by PHP.

It should be solve problem i guess:
`/HTTP\/\d(?:\.\d)?\s+(\d+)\s+/`

Someone else getting the same error?

We are using SDK v. 5.6.3.

### PR DESCRIPTION
**Pull request by jestinas**
closes #1076

Fix for the issue mentioned above thread.

### CODE REVIEW

File: src/Facebook/Http/GraphRawResponse.php [hunk 1]
```diff
@@ -104,8 +104,10 @@ public function getHttpResponseCode()
      */
     public function setHttpResponseCodeFromHeader($rawResponseHeader)
     {
-        preg_match('|HTTP/\d\.\d\s+(\d+)\s+.*|', $rawResponseHeader, $match);
-        $this->httpResponseCode = (int)$match[1];
+       /* preg_match('|HTTP/\d\.\d\s+(\d+)\s+.*|', $rawResponseHeader, $match);
+        $this->httpResponseCode = (int)$match[1];*/
```
Comment (yguedidi) [inline src/Facebook/Http/GraphRawResponse.php]: comment must be removed

**Review by yguedidi — CHANGES_REQUESTED** [review]
Please also add a test and update the changelog, thanks.

**Comment by yguedidi** [comment]
@jestinas thank you for your contribution! I did the review :)

**Comment by yguedidi** [comment]
Sorry @jestinas, I'm closing this in favor of #1079 as it was open before. Can't wait to see you contribute again!

**Comment by jestinas** [comment]
@yguedidi Cool brother. :)

**Pull request closed without merging**